### ISSUE TITLE
"/test" command is not registered.

### ISSUE THREAD

**Issue by Mickey42302** [pl3xgaming/Purpur#1504]
### Spark link

https://spark.lucko.me/D62uie0Ye0

### Expected behavior

When the "register-minecraft-debug-commands" setting is set to "true", this should register all of the hidden debugging commands.

### Observed/Actual behavior

The "/test" command is not registered when "register-minecraft-debug-commands" is set to "true". The other debugging commands are registered, however.

### Steps/models to reproduce

In the Purpur configuration, set "register-minecraft-debug-commands" to "true". When you attempt to run "/test", the server will behave as if it doesn't exist due to it not being registered. The rest of the debugging commands (such as "/raid", "/warden_spawn_tracker", and "/serverpack") will be registered, however.

### Purpur version

Current: git-Purpur-2309 (MC: 1.21.1)
Previous: git-Purpur-2308 (MC: 1.21.1)

### Agreements

- [X] I am running the latest version of Purpur available from https://purpurmc.org/downloads.
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [ ] I ticked all the boxes without actually reading them
- [X] My version of Minecraft is supported by Purpur.

**Comment by granny** [comment]
Correct. I've purposefully excluded that command from being registered because it's been annoying trying to get it to work without it instantly kicking the player that has permission to use it. It's pretty low on my priority list but I'll keep this issue open in case I end up revisiting it.

**Comment by granny** [comment]
The `resetchunks` command no longer exists since the release of version 1.21 [1.21-pre1] (it existed from 1.18 [21w37a] to 1.20.6).

For future reference please either create a new issue tracker if you find a command that isn't registered or add a comment with new information. (I don't get a notification when you edit the title or issue description).

**Comment by Mickey42302** [comment]
> The `resetchunks` command no longer exists since the release of version 1.21 [1.21-pre1] (it existed from 1.18 [21w37a] to 1.20.6).
> 
> For future reference please either create a new issue tracker if you find a command that isn't registered or add a comment with new information. (I don't get a notification when you edit the title or issue description).

Alright, I'll keep that in mind. I've updated the title/description as well since "/resetchunks" is no longer available.

**Comment by Onako2** [comment]
Can be closed once 1.21.5 releases because it adds the test command directly into the game

**Comment by granny** [comment]
closing this issue since 1.21.5 has released with the command available to everyone.
